packages/cozy-stack-client/src/mangoIndex.js

```diff
@@ -46,26 +46,61 @@ export const normalizeDesignDoc = designDoc => {
   return { id, _id: id, ...designDoc.doc }
 }
 
+/**
+ * Process a partial filter to generate a string key
+ *
+ * @param {object} condition - An object representing the partial filter or a sub-condition of the partial filter
+ * @returns {string} - The string key of the processed partial filter
+ */
+export const processPartialFilter = condition => {
```

Comment:
nit: `processPartialFilter` sounds like it will actually execute the partial filter, maybe it would be clearer with something like `makeKeyFromPartialFilter`